src/Analyzers/CSharp/CodeFixes/UseNullPropagation/CSharpUseNullPropagationCodeFixProvider.cs

```diff
@@ -32,6 +33,32 @@ public CSharpUseNullPropagationCodeFixProvider()
         {
         }
 
+        protected override bool TryGetBlock(SyntaxNode? statement, [NotNullWhen(true)] out StatementSyntax? block)
+        {
+            if (statement is BlockSyntax stetementBlock)
+            {
+                block = stetementBlock;
```

Comment:
Soryy, can you clarify? The `statement` name is already taken and I cannot assign `block = statement` directly without explicit cast because of type mismatch

Reply:
mispelling  stete not state  

sorry   on mobile